modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -5,40 +5,98 @@
-== Required vCenter account privileges
-
-To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
-
-A user requires the following privileges to install an {product-title} cluster:
-
-* Datastore
-** *Allocate space*
-** *Browse datastore*
-** *Low level file operations*
-** *Remove file*
-* Folder
-** *Create folder*
-** *Delete folder*
-* vSphere Tagging
-** All privileges
-* Network
-** *Assign network*
-* Resource
-** *Assign virtual machine to resource pool*
-* Profile-driven storage
-** All privileges
-* vApp
-** All privileges
-* Virtual machine
-** All privileges
-
-For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
+== Applying the required vCenter account privileges
+
+To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources.
+
+[NOTE]
+====
+It is recommended that you apply role access privileges globally to ensure that the required permissions are met. If permissions are not applied globally, you must determine the entity permissions. At minimum, a `Read-Only` role is required for the vCenter.
+====
+
+.Procedure
+. As a vSphere administrator, create a new custom role in your vCenter instance if you have not already done so. At minimum, the role must allow a user to read properties in both vCenter and all necessary vSphere entitites.
++
+If you do not assign any privileges to it, the role is created with `Read-Only` privileges.
+. As a vSphere administrator, apply the newly created `Read-Only` role to the vCenter user configured in {product-title}, using the parent entities shown in the following table:
++
+.Required role-based properties
+[cols="1a,1a,1a,1a",options="header"]
+|===
+
+|Role
+|Privileges
+|Entities
+|Propagate to children?
+
+|`Read-Only` (created by default)
+|`System.Anonymous`, `System.View`, `System.Read`
+|vCenter, Datacenter, Datastore Cluster, Datastore Storage Folder
+|No
+|===
++
+
+. Alternatively, as a non-vSphere administrator, apply the necessary privileges so that a user or group associated with a `Read-Only` role can install an {product-title} cluster, such as in the following example:
++
+.Example of vCenter access based on a custom role
+[cols="1a,5a",options="header"]
+|===
+
+|Component
+|Privileges
+
+|Datastore
+|* *Allocate space*
+* *Browse datastore*
+* *Low level file operations*
+* *Remove file*
+
+|Folder
+|* *Create folder*
+* *Delete folder*
+
+|vSphere Tagging
+|* All privileges
+
+|Host
+|* All privileges
```

Comment:
This was missing in the table prior to this PR but adding it here because I see it referenced here: https://github.com/openshift/installer/pull/4393/files#diff-8b34b91ae00a49ec13b47e1d2769dafab01e1aecf2f1e8844412e0337ebcddbd
Not sure what privileges are required, so went with "All" ¯\_(ツ)_/¯.

Reply:
@bobfuru thanks for catching this, but I think it should be the other way around: we should remove `host` from installer repo, not add it here.

Reply:
OK, removed `host` from this doc.